src/treemap.ts

```diff
@@ -55,11 +50,19 @@ export function isDOMNode(e: Element): boolean {
  */
 export interface Options {
   padding: [number, number, number, number];
-  caption?(node: Node): string;
+  lowerBound: number;
+  classList: ((node: Node) => string)[];
+  nodeColor: (node: Node) => NodeColor;
+  caption(node: Node): string;
```

Comment:
Did you mean to make this non-optional?

Reply:
Yes. As far as i can tell everywhere an `options` variable is typed it is done so as `Partial<Options>` which makes the optional field redundant.